### PR TITLE
Add common files for cypress test

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
The files common and store are added to cypress/views
These files are needed for other test execution.

Signed-off-by: Timothy Asir Jeyasingh <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@bipuladh , @vbnrh , Please review

File: cypress/views/common.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+export const commonFlows = {
+  navigateToOCS: (isOperatorsOpen = false) => {
+    const path = isOperatorsOpen ? ['Installed Operators'] : ['Operators', 'Installed Operators'];
+    cy.clickNavLink(path);
+    cy.byLegacyTestID('item-filter').type('ocs');
+    cy.byTestOperatorRow('OpenShift Container Storage').click();
+  },
+  checkAll: () => cy.get('input[name=check-all]'),
```
Comment (bipuladh) [inline cypress/views/common.ts:8]: This is obsolete.

File: cypress/views/store.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
+export enum Providers {
```
Comment (bipuladh) [inline cypress/views/store.ts:1]: This file is obsolete.

File: cypress/views/common.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      return;
+    }
+    if (retry <= 0) {
+      assert(false);
+      return;
+    }
+
+    commandPoll(cmd, expected, failOnNonZeroExit, retry - 1);
+  });
+};
```
Comment (bipuladh) [inline cypress/views/common.ts:29]: This is a small function and we can bring it along with it's usage.

**Review by bipuladh — COMMENTED** [review]
I think we can pretty much close this PR.

**Pull request closed without merging**